### PR TITLE
fix(filterPageUI): fixed edit and delete ui

### PR DESCRIPTION
**Pull request by simran142002**
### Functionality:
Fixed the ui of edit and delete in filter page

### Solution:
made changes in filter component

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve -o
firebase emulators:start

### CODE REVIEW

File: src/app/body/filter-page/edit-filter/edit-filter.component.css [hunk 1]
```diff
@@ -11,16 +11,109 @@
...
+.clr {
+    border: none;
+    color: var(--text);
+    background-color: var(--primary-bg);
+}
+
+.card {
+    background-color: var(--secondary-bg);
+    color: var(--text);
+    width: 24rem
```
Comment (Pendyala-thanvi) [inline src/app/body/filter-page/edit-filter/edit-filter.component.css:54]: Make the width responsive. Giving it in 'rem' will make it static and the card will not resize according to the screen or device.

Comment (simran142002) [reply src/app/body/filter-page/edit-filter/edit-filter.component.css:54]: okay I will make the changes

File: src/app/body/filter-page/edit-filter/edit-filter.component.css [hunk 2]
```diff
@@ -11,16 +11,109 @@
...
+
+
 #button {
     background-color: var(--primary-color);
     color: white;
 }
 
+.btn {
+    color: var(--text);
+}
```
Comment (abhishek-mishra019) [inline src/app/body/filter-page/edit-filter/edit-filter.component.css:32]: you are using .btn{} twice in line 20 and 30

File: src/app/body/filter-page/filter-page.component.css [hunk 3]
```diff
@@ -24,4 +34,4 @@
     background-color: var(--secondary-bg);
     color: var(--text);
     width: 24rem
```
Comment (abhishek-mishra019) [inline src/app/body/filter-page/filter-page.component.css:36]: do not use rem here, maybe define width in percentage

Comment (simran142002) [reply src/app/body/filter-page/filter-page.component.css:36]: okay I did not make this change but if you say I will add it in my pr

File: src/app/body/filter-page/edit-filter/edit-filter.component.css (hunk 2 above)
Comment (simran142002) [reply src/app/body/filter-page/edit-filter/edit-filter.component.css:32]: okay I will make the change

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**